sec2_the_ruby_programming_language/2.4_comparison.rb

```diff
@@ -0,0 +1,16 @@
+puts 10 == 9            #false
+puts 10 == "10".to_i    #true
+puts 10 == "10".to_f    #true
+puts 10 != 9            #true
+puts "hello" != "bye"   #true
+puts "hello" != "hello" #false
+puts "hello" == "hello" #true
+puts 100>99             #true
+puts 100>=100           #true
+puts 100>= 99           #true
+puts 100>= 101          #false
```

Comment:
can you keep your code consistent?
other places you have spacing around the equalities. here you don't have. choose one and stick with it